scripts/hid_configurator/led_stream.py

```diff
@@ -65,5 +75,187 @@ def send_continuous_led_stream(dev, recipient, DEVICE_CONFIG, led_id, freq, subs
     print('LED stream ended')
 
 
+def peak_to_hue(peak):
```

Comment:
why peak_to_hue?

Reply:
This is function that converts peak value calculated from audio signal to LED hue.